Tasks/NpmAuthenticateV0/package.json

```diff
@@ -17,7 +17,13 @@
   },
   "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
   "dependencies": {
-    "azure-pipelines-task-lib": "2.8.0",
-    "packaging-common": "file:../../_build/Tasks/Common/packaging-common-1.0.1.tgz"
+    "azure-pipelines-task-lib": "^3.1.10",
+    "azure-pipelines-tasks-packaging-common-v3": "^3.200.0",
```

Comment:
Please bump this version to latest (3.206.0)

Reply:
bumped "azure-pipelines-tasks-packaging-common-v3 to "^3.206.0"